axes/locale/pl/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,98 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+# 
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-06-11 23:36+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=4; plural=(n==1 ? 0 : (n%10>=2 && n%10<=4) && (n"
+"%100<12 || n%100>14) ? 1 : n!=1 && (n%10>=0 && n%10<=1) || (n%10>=5 && n"
+"%10<=9) || (n%100>=12 && n%100<=14) ? 2 : 3);\n"
+
+#: .\axes\admin.py:26
+msgid "Form Data"
+msgstr "Dane formularza"
+
+#: .\axes\admin.py:27 .\axes\admin.py:64
+msgid "Meta Data"
+msgstr "Metadane"
+
+#: .\axes\conf.py:89
+msgid "Account locked: too many login attempts. Please try again later."
+msgstr ""
+"Konto zablokowane: zbyt wiele prób logowania. Spróbuj ponownie później."
+
+#: .\axes\conf.py:97
+msgid ""
+"Account locked: too many login attempts. Contact an admin to unlock your "
+"account."
+msgstr ""
+"Konto zablokowane: zbyt wiele prób logowania. Skontaktuj się z "
+"administratorem, aby odblokować swoje konto."
+
+#: .\axes\models.py:6
+msgid "User Agent"
+msgstr ""
```

Comment:
I think there is no official translation for "User Agent" in polish language. "Agent użytkownika" is not directly connected to thinking about "browser" or any other device making HTTP connection - it will make confusion because it ephasizes the word "Agent" which has a specific meaning in polish - i think we (polish people) use "User Agent" as a proper name and do not translate it.